src/dnlib.csproj

```diff
@@ -15,6 +15,8 @@
     <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\dnlib.snk</AssemblyOriginatorKeyFile>
+    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+    <FrameworkPathOverride Condition="'$(OS)' != 'Windows_NT'">/usr/lib/mono/2.0-api/</FrameworkPathOverride>
```

Comment:
Can you confirm if this fix is for [this bug](https://github.com/dotnet/sdk/issues/335) ? If so you can add the hard coded paths (with Path.Exists checks) since it appears that MS does the same thing. But add a comment in the csproj file pointint to the issue.